coding/wml-v20/src/main/java/org/n52/sos/aquarius/OmObservationMapper.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (C) 2012-2015 Aquatic Informatics
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation.
+ *
+ * If the program is linked with libraries which are licensed under one of
+ * the following licenses, the combination of the program with the linked
+ * library is not considered a "derivative work" of the program:
+ *
+ *     - Apache License, version 2.0
+ *     - Apache Software License, version 1.0
+ *     - GNU Lesser General Public License, version 3
+ *     - Mozilla Public License, versions 1.0, 1.1 and 2.0
+ *     - Common Development and Distribution License (CDDL), version 1.0
+ *
+ * Therefore the distribution of the program linked with libraries licensed
+ * under the aforementioned licenses, is permitted by the copyright holders
+ * if the distribution is compliant with both the GNU General Public
+ * License version 2 and the aforementioned licenses.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ */
+package org.n52.sos.aquarius;
+
+import org.n52.sos.ogc.om.OmObservation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class OmObservationMapper {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(OmObservationMapper.class);
+    private static final String SEPARATOR_CHARACTER = "_";
+    private static final String DEFAULT_AQUARIUS_INTERPOLATION_TYPE = InterpolationTypeMapper.INSTANTANEOUS;
+    private static final String BASE_LOGGER_INTERPOLATION_MAPPING_WARNING =
+            "Could not determine aquarius interpolation type";
+    private final InterpolationTypeMapper interpolationTypeMapper;
+    private final ObservedPropertyMapper observedPropertyMapper;
+
+    public OmObservationMapper(InterpolationTypeMapper interpolationTypeMapper,
+            ObservedPropertyMapper observedPropertyMapper) {
+        this.interpolationTypeMapper = interpolationTypeMapper;
+        this.observedPropertyMapper = observedPropertyMapper;
+    }
+
+    public String toInterpolationTypeTitle(OmObservation observation) {
```

Comment:
These two methods look very similar. Stuff could be extracted from these (fine in a separate PR).